doc/api/async_hooks.md

```diff
@@ -973,6 +973,10 @@ chains. It allows storing data throughout the lifetime of a web request
 or any other asynchronous duration. It is similar to thread-local storage
 in other languages.
 
+While you can implement your own implementation on top of the `async_hooks`
```

Comment:
```suggestion
While you can build your own implementation on top of the `async_hooks`
```

Reply:
Personally, I prefer `implement` over `build`; `build` is more vague and more abstract whereas `implement` has a concrete feel to it. As a reader, when I read `build` I feel like it's going to be something complicated involving a lot of things, whereas when I read `implement` I feel like it's going to be a couple of 100 LOCs.

Reply:
Ah I got it, you don't want `implement` because it occurs twice in `implement your own implementation`. I just replaced `implement` with `create`. I guess that resolves the issue you were pointing to.

Reply:
Thanks.